src/loaders/vorbis.c

```diff
@@ -1391,7 +1396,6 @@ static uint8 get8(vorb *z)
    return c;
    }
    #endif
-   return 0;  /* silence warnings */
```

Comment:
Are you sure about these? They come from [here](https://github.com/libxmp/libxmp/commit/bf129468db9cc0840e3ea61f50e54f07dcf4dcb5), and are likely for old compilers (guessing circa GCC 4.0).

(The `calloc`s look like an old Claudio thing with little commit documentation. Since I'm fuzzing the seek functions currently I will watch to see if those cause any unusual behavior after merge.)

Reply:
I thought watocm was the only oddball here and had added warning disablement to makefike,
but you turned out to be right. gcc-4.0 emits the following at -O3. Put them back now.

````
src/loaders/vorbis.c: In function 'vorbis_find_page':
src/loaders/vorbis.c:4660: warning: control may reach end of non-void function 'get8' being inlined
src/loaders/vorbis.c:4662: warning: control may reach end of non-void function 'libxmp_vorbis_get_file_offset' being inlined
src/loaders/vorbis.c:4714: warning: control may reach end of non-void function 'set_file_offset' being inlined
src/loaders/vorbis.c: In function 'libxmp_vorbis_stream_length_in_samples':
src/loaders/vorbis.c:5098: warning: control may reach end of non-void function 'getn' being inlined
````